layers/+distribution/spacemacs-base/keybindings.el

```diff
@@ -341,6 +341,7 @@
   "xa:" 'spacemacs/align-repeat-colon
   "xa;" 'spacemacs/align-repeat-semicolon
   "xa=" 'spacemacs/align-repeat-equal
+  "xa\\" 'spacemacs/align-repeat-backslash
```

Comment:
I didn't align this to the rest of the options listed here for fear that it would obscure the change but can do so if desired.